dm/pkg/binlog/reader/tcp_test.go

```diff
@@ -21,130 +21,131 @@ import (
 	gmysql "github.com/go-mysql-org/go-mysql/mysql"
 	"github.com/go-mysql-org/go-mysql/replication"
 	_ "github.com/go-sql-driver/mysql"
-	. "github.com/pingcap/check"
 	"github.com/pingcap/failpoint"
 	"github.com/pingcap/tiflow/dm/pkg/binlog/common"
 	"github.com/pingcap/tiflow/dm/pkg/gtid"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
 )
 
 var (
-	_         = Suite(&testTCPReaderSuite{})
 	flavor    = gmysql.MySQLFlavor
 	serverIDs = []uint32{3251, 3252}
 )
 
-func TestSuite(t *testing.T) {
-	TestingT(t)
+func TestTCPReaderSuite(t *testing.T) {
+	suite.Run(t, new(testTCPReaderSuite))
 }
 
-type testTCPReaderSuite struct{}
+type testTCPReaderSuite struct {
+	suite.Suite
+}
 
-func (t *testTCPReaderSuite) SetUpSuite(c *C) {
-	c.Assert(failpoint.Enable("github.com/pingcap/tiflow/dm/pkg/binlog/reader/MockTCPReaderStartSyncByPos", "return(true)"), IsNil)
-	c.Assert(failpoint.Enable("github.com/pingcap/tiflow/dm/pkg/binlog/reader/MockTCPReaderStartSyncByGTID", "return(true)"), IsNil)
-	c.Assert(failpoint.Enable("github.com/pingcap/tiflow/dm/pkg/binlog/reader/MockTCPReaderClose", "return(true)"), IsNil)
-	c.Assert(failpoint.Enable("github.com/pingcap/tiflow/dm/pkg/binlog/reader/MockTCPReaderGetEvent", "return(true)"), IsNil)
-	c.Assert(failpoint.Enable("github.com/pingcap/tiflow/dm/pkg/binlog/reader/MockTCPReaderStatus", "return(true)"), IsNil)
+func (t *testTCPReaderSuite) SetUpSuite() {
```

Comment:
should be `SetupSuite`

you can check the doc about this interface

https://pkg.go.dev/github.com/stretchr/testify/suite#SetupAllSuite

https://pkg.go.dev/github.com/stretchr/testify/suite#TearDownAllSuite